### PR TITLE
Archive & Job Status Tab

### PR DESCRIPTION
**Pull request by vermadeveloper**

### CODE REVIEW

File: application/migrations/20190610151320_update_jobs_table.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?php
```
Comment (ankur2194) [inline application/migrations/20190610151320_update_jobs_table.php]: Change the date on this file name to today or yesterday.

File: application/migrations/20191806105211_update_jobs_status_table.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?php
```
Comment (ankur2194) [inline application/migrations/20191806105211_update_jobs_status_table.php]: Correct the date format.
Also make sure the date on this file name is of today or yesterday.

File: application/migrations/20191806105211_update_jobs_status_table.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        ];
+
+        $this->dbforge->modify_column('jobs_status', $fields);
+    }
+
+    public function down()
+    {
+        $fields = [
+            'start_at' => [
+                'type' => 'DATETIME'
```
Comment (ankur2194) [inline application/migrations/20191806105211_update_jobs_status_table.php]: What are the changes here?

File: application/migrations/20191806105211_update_jobs_status_table.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            ],
+            'close_at' => [
+                'type' => 'DATETIME'
```
Comment (ankur2194) [inline application/migrations/20191806105211_update_jobs_status_table.php]: What are the changes here?

File: application/migrations/20191806105211_update_jobs_status_table.php (hunk 3 above)
Comment (vermadeveloper) [reply application/migrations/20191806105211_update_jobs_status_table.php]: Job Signed Date

File: application/migrations/20191806105211_update_jobs_status_table.php (hunk 4 above)
Comment (vermadeveloper) [reply application/migrations/20191806105211_update_jobs_status_table.php]: Job Closing date

File: application/migrations/20191806105211_update_jobs_status_table.php (hunk 2 above)
Comment (vermadeveloper) [reply application/migrations/20191806105211_update_jobs_status_table.php]: done

File: application/migrations/20190610151320_update_jobs_table.php (hunk 1 above)
Comment (vermadeveloper) [reply application/migrations/20190610151320_update_jobs_table.php]: done

File: application/migrations/20191806105211_update_jobs_status_table.php (hunk 3 above)
Comment (ankur2194) [reply application/migrations/20191806105211_update_jobs_status_table.php]: Why Up and Down both has same code to execute.

File: application/migrations/20191806105211_update_jobs_status_table.php (hunk 4 above)
Comment (ankur2194) [reply application/migrations/20191806105211_update_jobs_status_table.php]: Why Up and Down both has same code to execute?

Comment (vermadeveloper) [reply application/migrations/20191806105211_update_jobs_status_table.php]: My mistake. Now check

File: application/migrations/20191806105211_update_jobs_status_table.php (hunk 3 above)
Comment (vermadeveloper) [reply application/migrations/20191806105211_update_jobs_status_table.php]: My mistake. Now check

File: application/migrations/20190618105211_update_jobs_status_table.php [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        $fields = [
+            'start_at' => [
+                'type' => 'DATETIME'
+            ],
+            'close_at' => [
+                'type' => 'DATETIME'
+            ],
+        ];
+        $this->dbforge->drop_column('jobs_status', $fields);
```
Comment (ankur2194) [inline application/migrations/20190618105211_update_jobs_status_table.php]: Do you think this is correct?

File: application/migrations/20190618105212_update_jobs_table.php [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+    public function down()
+    {
+        $fields = [
+            'zip' => [
+                'type' => 'int',
+                'constraint' => 5
+            ],
+        ];
+        $this->dbforge->modify_column('jobs', $fields);
```
Comment (ankur2194) [inline application/migrations/20190618105212_update_jobs_table.php]: Do you think this is correct?

Comment (vermadeveloper) [reply application/migrations/20190618105212_update_jobs_table.php]: Yes this is correct. Anything wrong?

Comment (ankur2194) [reply application/migrations/20190618105212_update_jobs_table.php]: Yes Sir.
You just corrected the another file, why not corrected this one?

**Pull request merged**